### PR TITLE
Ralph3: blade_server collect plugin

### PR DESCRIPTION
**Pull request by mkurek**

### CODE REVIEW

File: src/ralph_scrooge/plugins/collect/ralph3_blade_server.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    """
+    Updates single Blade Server usage type record
+    """
+    try:
+        asset_info = AssetInfo.objects.get(ralph3_asset_id=data['id'])
+        daily_asset_info = asset_info.dailyassetinfo_set.get(date=date)
+        return update_usage(
+            daily_asset_info,
+            date,
+            1,
```
Comment (xor-xor) [inline src/ralph_scrooge/plugins/collect/ralph3_blade_server.py:67]: What about adding here a short comment explaining what this `1` means..?

**Comment by xor-xor** [comment]
:ok:

**Comment by szok** [comment]
:ok:

**Comment by szok** [comment]
:ok:

**Pull request merged**